downstairs/src/extent.rs

```diff
@@ -669,6 +676,12 @@ pub(crate) fn move_replacement_extent<P: AsRef<Path>>(
                 e
             );
         }
+        info!(
```

Comment:
Do you want to keep these new log messages around?

Reply:
Probably not.  Now that I know where the bug was, I can remove them.
Done in [7bef555](https://github.com/oxidecomputer/crucible/commit/7bef55560bd1a345855ca77b6c296616ee66e0fe)